<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络分析软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/qrob.png</normaloff>:/res/qrob.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{background:white;}
QLabel{ font-size:18px;}
QToolButton{border:none;    font-size:18px;}
QPushButton{
    color:#FFFFFF;
    font-size:16px;
    border-radius:4px;
    min-height:30px ;
    padding-left:15px;
    padding-right:15px;
    background:#409EFF;
}
QPushButton:pressed{
background:#3a8ee6;
}
QLineEdit{border-radius:2px;border:1px solid gray;}
QPushButton:disabled{
background:#A0CFFF;
}

    </string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="DataTable" name="tableWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>300</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0" colspan="4">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="DataTree" name="treeWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
         <widget class="QTextBrowser" name="textBrowser"/>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="toolButton">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>网卡信息:</string>
         </property>
         <property name="icon">
          <iconset resource="../res.qrc">
           <normaloff>:/res/icon_net.svg</normaloff>:/res/icon_net.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_net">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_filter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>过滤器:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_filter">
         <property name="text">
          <string>过滤</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="anction_start"/>
   <addaction name="separator"/>
   <addaction name="action_select"/>
   <addaction name="separator"/>
   <addaction name="action_ansy"/>
   <addaction name="separator"/>
   <addaction name="action_clear"/>
   <addaction name="separator"/>
   <addaction name="action_about"/>
  </widget>
  <action name="action_select">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/icon_select.svg</normaloff>:/res/icon_select.svg</iconset>
   </property>
   <property name="text">
    <string>选择网卡</string>
   </property>
  </action>
  <action name="action_ansy">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/icon_ansy.svg</normaloff>:/res/icon_ansy.svg</iconset>
   </property>
   <property name="text">
    <string>流量统计</string>
   </property>
  </action>
  <action name="anction_start">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/icon_start.svg</normaloff>:/res/icon_start.svg</iconset>
   </property>
   <property name="text">
    <string>开始抓包</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_clear">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/icon_clear.svg</normaloff>:/res/icon_clear.svg</iconset>
   </property>
   <property name="text">
    <string>清除信息</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/res/icon_about.svg</normaloff>:/res/icon_about.svg</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTable</class>
   <extends>QTableWidget</extends>
   <header>actor/datatable.h</header>
  </customwidget>
  <customwidget>
   <class>DataTree</class>
   <extends>QTreeWidget</extends>
   <header>actor/datatree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
